win_configure/msi/wix/prep.bat

```diff
@@ -0,0 +1,257 @@
+@echo off
+REM This script will generate the componentgroup directory structure
+
+REM parsing command line arguments
+set argC=0
+for %%x in (%*) do Set /A argC+=1
+
+if %argC% LSS 1 (
```

Comment:
Rather than using for loop to count number of argument and then checking it we can do as follow:
```bat
if not "%1" == "" (
   exit 1
)
```